np.sh

```diff
@@ -1,5 +1,10 @@
 #!/usr/bin/env sh
 
+if test "6" == "$(node -v | sed -e "s#v\([0-9]*\)\.\([0-9]*\)\.\([0-9]*\)#\1#")"; then
```

Comment:
The `6` test should be on the other side. The regex only need to capture the first number.